rllib/BUILD

```diff
@@ -2571,7 +2571,7 @@ py_test(
 py_test(
     name = "examples/rnnsac_stateless_cartpole",
     tags = ["team:ml", "gpu"],
-    size = "large",
+    size = "enormous",
```

Comment:
I think across OSS we're avoiding the usage of "enormous". Can we change the test somehow?

Reply:
I've added gpu functionality internally to this launcher -- I'm hoping that its not necessary to keep this as is. Perhaps we can fix the seed of this experiment, that way there is deterministic output across ci runs

Reply:
I fixed the seed, and reduced the number of training timesteps for this example.